packages/react-composites/src/composites/CallComposite/CallScreen.tsx

```diff
@@ -172,13 +170,13 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
               </>
             )}
           </Stack.Item>
-          {props.visualElements.showCallControls && (
+          {props?.hiddenElements?.callControls !== true && (
```

Comment:
This can just be `props?.hiddenElements?.callControls || ( ... )`. Yay for falsey default values!

Reply:
Thanks, totally forgot about that shorthand. I would prefer leaving it explicit right now since it is a little hard to read. :D